### ISSUE TITLE
[5.0.0.0-beta] CodeCoverage not available

### ISSUE THREAD

**Issue by mrboring** [pester/Pester#1464]
## 1. General summary of the issue

The CodeCoverage parameter is not available.

## 2. Describe Your Environment

```
Pester version     : 5.0.0 C:\Users\<REMOVED>\Documents\PowerShell\Modules\Pester\5.0.0\Pester.psd1
PowerShell version : 7.0.0
OS version         : Microsoft Windows NT 10.0.18363.0


> get-module pester -ListAvailable | ? Version -eq '5.0.0'

    Directory: C:\Users\Alan\Documents\PowerShell\Modules

ModuleType Version    PreRelease Name                                PSEdition ExportedCommands
---------- -------    ---------- ----                                --------- ----------------
Script     5.0.0      beta       Pester                              Desk      {Describe, Context, It, Should…}


Name                           Value
----                           -----
PSVersion                      7.0.0
PSEdition                      Core
GitCommitId                    7.0.0
OS                             Microsoft Windows 10.0.18363
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

## 3. Expected Behavior

I expected the `CodeCoverage` parameter to be available.

## 4.Current Behavior

I wanted to use CodeCoverage. When I type `Invoke-Pester -`, then do `Shift+Space` I get:

```
> Invoke-Pester -Path
Path                 ExcludeTag           Configuration        ErrorAction          ErrorVariable        OutVariable
ExcludePath          CI                   Verbose              WarningAction        WarningVariable      OutBuffer
Tag                  Output               Debug                InformationAction    InformationVariable  PipelineVariable
```

No sign of `CodeCoverage`.

## 6. Context

I'm trying to check code coverage for my Pester tests.

**Comment by nohwnd** [comment]
Hello, thanks for giving it a try. Right now the CodeCoverage is available only via the Advanced api, and by default it should pick up all the .ps1 files (not including tests) that are in the same folder as a  tests.ps1 file, and then save the result in coverage.xml in the current folder. Here an example from the linked readme. 

```powershell
$configuration = [PesterConfiguration]@{
    Run = @{
        Path = 'C:\projects\tst'
    }
    Filter = @{
        Tag = 'Acceptance'
        ExcludeTag = 'WindowsOnly'
    }
    CodeCoverage = @{
        Enable = $true
    }
} 
```

Code coverage is probably a bit buggy right now, so please don't hesitate to ask for more info here, or report bugs 🙂

https://github.com/pester/Pester/tree/v5.0#simple-and-advanced-interface

**Comment by nohwnd** [comment]
This should be addressed as part of #1265, in the meantime the parameter was added as part of the legacy parameter set for backwards compatibility.
